erpnext/public/js/controllers/transaction.js

```diff
@@ -909,6 +909,18 @@ erpnext.TransactionController = class TransactionController extends erpnext.taxe
 				}
 			});
 
+			frappe.call({
```

Comment:
@noahjacob a method to fetch shipping address is already called on line no 896, can you check why that is not working and fix that instead of fetching shipping address again

Made a similar fix for purchase invoice. The company trigger itself was not getting called. Maybe this might help
https://github.com/frappe/erpnext/pull/26100

Reply:
![image](https://user-images.githubusercontent.com/43572428/122755943-590dfd00-d2b3-11eb-9331-e3b5dc4d15f2.png)

@deepeshgarg007 it's not fetching because of line 279. The condition can be moved elsewhere? Maybe in the callback on line 297?

Reply:
If this condition is the reason then it means that the Purchase Document is an internal transfer document. The address setting in this case is done on the server-side. Check this function in `sales_invoice.py`. Please check first why the address is not getting updated from here first

![image](https://user-images.githubusercontent.com/42651287/122914437-43aed680-d378-11eb-9f1e-6842d7004d0c.png)

Reply:
@deepeshgarg007  this is a not condition 
`if (!(frm.doc.inter_com_order_reference || frm.doc.internal_invoice_reference ||
frm.doc.internal_order_reference)) {`
So this checks if it is not an internal transfer right. I tested it with a non-internal transfer and it returns the callback.

Reply:
@noahjacob please remove the `!` then and let it return the callback only in case of internal docs.
Don't exactly remember the reason for adding the not sign.

Removing that will solve the issue, right?

Reply:
Yes, that should solve the issue. I'll also rename inter_com_order_reference. The fieldname is inter_company_order_reference in the doctype.